build.sbt

```diff
@@ -166,6 +166,23 @@ lazy val circe = crossProject(JSPlatform, JVMPlatform)
 lazy val circeJVM = circe.jvm
 lazy val circeJS  = circe.js
 
+lazy val akkaJVM = project
```

Comment:
I think this subproject needs to also occur in the aggregation of the root project, not? I think your tests are currently not run from within the ci (have to take a second look though…)

Reply:
Right. I fixed it.